cdap-docs/examples-manual/source/examples/index.rst

```diff
@@ -23,7 +23,7 @@ Examples
    Spark K-Means <spark-k-means>
    Spark Page Rank <spark-page-rank>
    Web Analytics <web-analytics>
-
+   Stream Conversion <stream-conversion>
```

Comment:
Should this (could this?) go above "Web Analytics"? Or are there categories we should order or group these in?
The order of this list should match the order of discussion below. It is the order they appear in the TOC and in the left sidebar.